### DISCUSSION TITLE
--

### DISCUSSION THREAD

**Discussion by cyberrrr** [Rapptz/discord.py#6334] [General] [answered]
--

**Comment by daggy1234** [comment, marked as answer]
https://cdn.discordapp.com/attachments/381963689470984203/382387651162144779/unknown.png
Also see: https://support.discordapp.com/hc/en-us/articles/115002192352-Automated-user-accounts-self-bots-

They are against discord's Terms of service, so I highly recommend not using a selfbot

**Reply by JavascriptLearner815** [reply]
> Who cares about the rules

Us.

**Reply by deleteduse** [reply]
> > Who cares about the rules
> 
> Us.

who is us lmaooo

**Reply by JavascriptLearner815** [reply]
> > > Who cares about the rules
> > 
> > 
> > Us.
> 
> who is us lmaooo

Not.
